env_docker.go

```diff
@@ -129,10 +130,10 @@ func New(opts ...EnvironmentOption) (_ *DockerEnvironment, err error) {
 		return nil, errors.Wrapf(err, "create docker network '%s'", d.networkName)
 	}
 
-	switch runtime.GOOS {
-	case "darwin":
-		d.hostAddr = dockerMacOSGatewayAddr
-	default:
+	switch e2einteractive.HostOSPlatform() {
+	case "darwin", "WSL2":
+		d.hostAddr = dockerGatewayAddr
+	case "linux":
```

Comment:
Should we fail on default? or use Linux?

Reply:
Good question. I think we can try Linux on default and if it fails in this case: panic. 😱